go/mysql/datetime/format_test.go

```diff
@@ -45,7 +45,7 @@ func TestFormattingFromMySQL(t *testing.T) {
 
 	for _, tc := range cases {
 		t.Run(tc.timestamp, func(t *testing.T) {
-			dt, ok := ParseDateTime(tc.timestamp)
+			dt, _, ok := ParseDateTime(tc.timestamp, -1)
```

Comment:
IMO it's worth adding a consts in the datetime package for -1,3,6... so that it's more obvious what the mean. Looks like you already did do that at least for `DefaultPrecision`, we're just not using them everywhere. The other ones are more obvious, but IMO still worth adding:
```
dt, _, ok := ParseDateTime(tc.timestamp, DefaultPrecision)

dt, _, ok := ParseDateTime(tc.timestamp, SecondPrecision)

dt, _, ok := ParseDateTime(tc.timestamp, MillisecondPrecision)

dt, _, ok := ParseDateTime(tc.timestamp, MicrosecondPrecision)

dt, _, ok := ParseDateTime(tc.timestamp, NanosecondPrecision)
```

Don't feel strongly about that, so up to you.